ddtrace_gevent_check.py

```diff
@@ -1,13 +0,0 @@
-import sys
-import warnings
-
-
-def gevent_patch_all(event):
-    if "ddtrace" in sys.modules:
-        warnings.warn(
-            "Loading ddtrace before using gevent monkey patching is not supported "
```

Comment:
do we need a warning for the opposite of this?

Reply:
It would be nice to have but we would need a way to find out if gevent-s `patch_all` was called without causing side-effects.

Reply:
🤔 yeah... probably would need to check if gevent is already loaded in `sys.modules`, and if it is, we can check if patching has been called yet, all before calling module cloning ?

But that doesn't help the case when just doing `import ddtrace`

Reply:
@brettlangdon I've suggested a fix for this in https://github.com/DataDog/dd-trace-py/pull/5105/commits/1040f1c2385e7a4f0e2c8a4fa0baf18fa3341846, please let me know if that addresses your concern.